examples/complex.json

```diff
@@ -12,6 +12,12 @@
       "version": "1.6.0",
       "targetDir": "examples"
     },
+    "system_uri": {
```

Comment:
See my previous comment about the examples, since this is an independent project to be used by any other it's better to not tight the examples to any of our other projects.